packages/rest/src/keys.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CoreBindings} from '@loopback/core';
+
+export namespace RestBindings {
+  // RestServer-specific bindings
+  export const CONFIG = `components.RestServer.config`;
+  export const PORT = 'http.port';
+  export const HANDLER = 'http.handler';
+
+  export const API_SPEC = 'components.RestServer.apiSpec';
+  export const SEQUENCE = 'sequence';
```

Comment:
Since sequence is transport-specific, this should say `components.RestServer.sequence`.

I would personally choose a shorter prefix `rest.` instead of `components.RestServer.`